packages/kbn-apm-synthtrace-client/src/lib/apm/apm_fields.ts

```diff
@@ -94,6 +95,7 @@ export type ApmFields = Fields<{
     'error.type': string;
     'event.ingested': number;
     'event.name': string;
+    'event.kind': string;
```

Comment:
Is this a new field for mobile? What does it represent?

Reply:
I was looking for ways to narrow down generic logs to only the ones we send as "events" using OTel's event API and I noticed that the logs I was targeting have this `event.kind` field set to `event`, so that's why I used it. It seems to be an [ECS field](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html). However, I'm not sure if it's the best way to do what I needed, I recall @bryce-b mentioning that another alternative could be to use the `event.action` attr instead, which OTel sends as `event.name`, and we could filter it to this particular event name, which is `lifecycle`. Although things started to fail once I did the change so I reverted it back, but if that's a better option, I can spend some more time on it until it works.

Reply:
thanks for the clarification. Can you reach out to the apm-server team to verify it?

Reply:
They mentioned that `event.kind` shouldn't be used for this use case, so I tried to replace it with `event.action` equals to `lifecycle` but doing so broke the tests in `mobile_location_stats.spec.ts`. I tried a couple of things to make them work with `event.action` without luck unfortunately. So in the end, I just removed all filters in this function, so that the only filters applied to the query would be the ones passed by the caller of this `logEventSearch` function. Which currently is only done in [here](https://github.com/elastic/kibana/pull/168925/files#diff-9c72e8d3fa1c355a012177beef983e912957c0015f139d5dddc1d708df2ec1a1R70) where the `labels.lifecycle_state` term is passed as filter.